src/lib/project-saver-hoc.jsx

```diff
@@ -37,7 +37,16 @@ import {
  */
 const ProjectSaverHOC = function (WrappedComponent) {
     class ProjectSaverComponent extends React.Component {
+        constructor (props) {
+            super(props);
+            this.state = {
+                autoSaveTimeoutId: null
```

Comment:
It just occurred to me that if this is in local state, it will get reset if we unmount/mount. Maybe we should keep it in redux to avoid that — it would have implications for switching language or swapping between player/editor mode.

Reply:
Hm. Let me try making that change.

Reply:
Take a look